iam.go

```diff
@@ -57,6 +59,13 @@ type Client struct {
 
 	// S3Credentials instance is used to make requests against Selectel IAM API and manage S3 Credentials.
 	S3Credentials *s3credentials.Service
+
+	// Federations instance is used to make requests against Selectel IAM API and manage Federations.
+	Federations *saml.Service
```

Comment:
This field represents SAML federations only, it seems that it would be better to reflect it in the field's name.
Same for certificates.

Reply:
Added prefix `SAML` to this field and changed comment above
Certificates was moved inside the `Federations` service